src/components/Shared/Navbar/index.module.scss

```
@@ -0,0 +1,20 @@
+.container {
+  margin: auto;
+  max-width: 78rem;
+  .navbar {
```

https://github.com/ahr-wtag/react-todo-app/pull/2#discussion_r1172177404

```suggestion
  max-width: 78rem;
  
  .navbar {
```